scss/utilities/mixins/organisms/_footer.scss

```diff
@@ -0,0 +1,36 @@
+@charset "UTF-8";
+
+//
+// Footer Mixins
+//
+// Mixins for the global footer.
+//
+// Style guide: Mixins.Organisms.Footer
+//
+
+@mixin footer-colors {
```

Comment:
Mixin name is too generic and the KSS comment does not do a good enough job at explaining what is going on. This also seems to be a pretty specific mixin without wide usage and could probably be avoided.